src/main/java/com/textmessenger/controller/DialogController.java

```diff
@@ -26,11 +26,13 @@ public DialogController(DialogService dialogService) {
   }
 
   @PostMapping("/user/{id}")
-  public ResponseEntity createDialog(@PathVariable("id") User user, @RequestBody Dialog dialog) {
-    List<User> users = dialog.getUsers();
-    users.add(user);
-    dialog.setUsers(users);
-    dialogService.createDialog(dialog);
+  public ResponseEntity createDialog(@PathVariable("id") Long user, @RequestBody User mainUser) {
+    System.out.println("===========================================================================");
+    System.out.println(user);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=127&issues=AWU5sjQMW2AOP0CkKi5n&open=AWU5sjQMW2AOP0CkKi5n)

[ISSUE_KEY]: AWU5sjQMW2AOP0CkKi5n